tasks/go.py

```diff
@@ -391,6 +411,40 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg in additional:
+        if pkg.startswith(GITHUB_DOMAIN):
+            url = "https://{}/master/".format(pkg.replace(GITHUB_DOMAIN, GITHUB_RAW_DOMAIN))
+        else:
+            url = "https://{}".format(pkg)
```

Comment:
In general a package name will not give us its source code, take for example `golang.org/x/sys`. I feel like we should just raise an Exception and give up on this case, we can deal with it if we ever need it